tests/validation/tests/v3_api/test_deploy_clusters.py

```diff
@@ -60,6 +68,40 @@ def test_deploy_rke():
         env_details += cluster.name
         print("Successfully deployed {} with kubernetes version {}".format(
             cluster.name, k8s_version))
+        cluster_details[cluster.name] = {}
+        cluster_details[cluster.name]["rke"] = k8s_version
+
+
+@if_not_auto_deploy_rke_import
+def test_deploy_rke_import():
+    print("Deploying RKE import Clusters")
+    global env_details
+    global cluster_details
+    errors = []
+    # get rke k8s versions
+    rkecommand = "rke config --list-version -a"
+    print(rkecommand)
```

Comment:
Nit: remove this print since the command is printed in the following function call

Reply:
done